docs/dev/execution_configuration.md

```diff
@@ -42,44 +42,44 @@ var executionConfig = env.getConfig
 
 The following configuration options are available: (the default is bold)
 
-- **`enableClosureCleaner()`** / `disableClosureCleaner()`. The closure cleaner is enabled by default. The closure cleaner removes unneeded references to the surrounding class of anonymous functions inside Flink programs.
+- **`enableClosureCleaner()`** / **`disableClosureCleaner()`**. The closure cleaner is enabled by default. The closure cleaner removes unneeded references to the surrounding class of anonymous functions inside Flink programs.
```

Comment:
As noted just above the list, the bold option specifies what the default is, we we're losing information here.

Reply:
@zentol I wasn't aware of that convention in all honesty, thank you for pointing it out.